### PR TITLE
example of invocation for ppo

### PR DESCRIPTION
**Pull request by piotrmilos**

### CODE REVIEW

File: tensor2tensor/rl/trainer_model_free.py [hunk 1]
```diff
@@ -32,11 +43,38 @@
...
+    "environment_eval_spec",
+    gym_problems.standard_atari_env_eval_spec("PongNoFrameskip-v4"))
+
+  return hparams
 
 def main(_):
   hparams = trainer_lib.create_hparams(FLAGS.hparams_set, FLAGS.hparams)
-  rl_trainer_lib.train(hparams, FLAGS.problem, FLAGS.output_dir)
-
+  rl_trainer_lib.train(hparams, FLAGS.output_dir, FLAGS.output_dir)
```
Comment (afrozenator) [inline tensor2tensor/rl/trainer_model_free.py]: line accidentally changed?

Comment (piotrmilos) [reply tensor2tensor/rl/trainer_model_free.py]: It is a bug fix.

Comment (afrozenator) [reply tensor2tensor/rl/trainer_model_free.py]: Ah! Just looked at the method signature now.

**Comment by afrozenator** [comment]
Thanks @piotrmilos !

**Pull request merged**